Gemfile.lock

```diff
@@ -114,4 +116,4 @@ DEPENDENCIES
   rake
 
 BUNDLED WITH
-   2.2.9
+   2.2.23
```

Comment:
Upgrading to the latest bundler version as well - since this is a patch version I don't think there are any major changes here.